WordPressKit/PostServiceRemoteREST.m

```diff
@@ -432,7 +436,13 @@ - (RemotePost *)remotePostFromJSONDictionary:(NSDictionary *)jsonPost {
     post.tags = [self tagNamesFromJSONDictionary:jsonPost[@"tags"]];
 
     post.revisions = [jsonPost arrayForKey:@"revisions"];
-    
+
+    NSDictionary *autosave = jsonPost[@"meta"][@"data"][@"autosave"];
+    post.autosaveTitle = autosave[@"title"];
+    post.autosaveContent = autosave[@"content"];
+    post.autosaveExcerpt = autosave[@"excerpt"];
+    post.autosaveModifiedDate = [NSDate dateWithWordPressComJSONString:autosave[@"modified"]];
```

Comment:
As mentioned in https://github.com/wordpress-mobile/WordPress-iOS/pull/12781:

- The `s` should be capitalized. 
- Let's add the other available properties even if we don't need them. Since they come for free in here, it's more expensive to modify `WordPressKit` again and add them later if we find out we need them.

Reply:
What do you think about making a separate class for this? Just so the `RemotePost` is a bit closer to the API result. 

```swift 
class RemotePostAutoSave {
    let title: String?
    let content: String?
    ...
}
```

The WPiOS flat properties are fine since they are used in Core Data.

Reply:
> What do you think about making a separate class for this? Just so the RemotePost is a bit closer to the API result.

I agree - and with more attributes being mapped it's especially useful to keep them in a separate class.

Leaving `s` uncapitalized as per discussion: https://github.com/wordpress-mobile/WordPress-iOS/pull/12781#discussion_r341083406